build.gradle

```diff
@@ -55,7 +55,7 @@ dependencies {
     final javadocJDKFiles = files(((URLClassLoader) ToolProvider.getSystemToolClassLoader()).getURLs())
     compileOnly(javadocJDKFiles)
     testCompile(javadocJDKFiles)
-    compile 'org.freemarker:freemarker:2.3.18'
+    compile 'org.freemarker:freemarker:2.3.23'
```

Comment:
@magicDGS Upgrading seems like a good idea. Since it sounds like there are some incompatibilities introduced, have you by any chance tried this with the gatkDoc process ? @jonn-smith Have you by any any chance tried using this version ? I doubt there is anything that would prevent us from upgrading, but it would be nice to know if there is any fallout.